kuma/dashboards/views.py

```diff
@@ -130,3 +131,19 @@ def topic_lookup(request):
     data = json.dumps(topiclist)
     return HttpResponse(data,
                         content_type='application/json; charset=utf-8')
+
+
+@require_POST
+def submit_akismet_spam(request):
```

Comment:
The permissions checks could be replaced with a [@permission_required() decorator](https://docs.djangoproject.com/en/1.9/topics/auth/default/#the-permission-required-decorator):

```
@permission_required('wiki.add_revisionakismetsubmission')
```